src/pool.jl

```diff
@@ -6,36 +6,27 @@ using Logging
 
 ## allocation statistics
 
-# PPC doesn't support floating-point atomics, so use a ref (which behaves mostly similar)
-if Float64 <: Threads.AtomicTypes
-const MaybeAtomicFloat64 = Threads.Atomic{Float64}
-_add_total_time!(stats, time) = Threads.atomic_add!(stats.total_time, time)
-else
-const MaybeAtomicFloat64 = Base.RefValue{Float64}
-_add_total_time!(stats, time) = (stats.total_time[] += time)
-end
-
 mutable struct AllocStats
   alloc_count::Threads.Atomic{Int}
   alloc_bytes::Threads.Atomic{Int}
 
   free_count::Threads.Atomic{Int}
   free_bytes::Threads.Atomic{Int}
 
-  total_time::MaybeAtomicFloat64
+  total_time::Threads.Atomic{Float64}
```

Comment:
Can we use `@atomic total_time::Float64`?

Reply:
I guess we could update almost all of these then?